test/test_hpyerr.py

```diff
@@ -69,3 +69,36 @@ def test_HPyErr_Cleared(self):
         """)
         assert mod.f() is None
         assert sys.exc_info() == (None, None, None)
+
+    def check_exception_constant(self, cls):
```

Comment:
thank you for adding these tests, we really needed them!
I wonder whether it's better to just have a single test for all of them to save compilation time, but your choice

Reply:
I initially did try a single test, but I couldn't think of a good way to do `ctx->h_XXX` (where `XXX` can change) without recompiling.

Reply:
what about using a dict? E.g.:
```c
HPy h_dict = HPyDict_New(ctx);
HPy_SetAttr_s(ctx, h_dict, "ValueError", ctx->h_ValueError);
...
```

Then you can just call `mod.f('ValueError')` and check that it raises. The `HPy_SetAttr_s` lines can be either written manually or autogenerated; my personal preference would be to write them manually, I like to have tests as linear and easy to read as possible.

Reply:
Ah, so type them all into the same module. Let me do that in the PR with all the exceptions I am working on now.